### PR TITLE
Add cheatsheet for special phx-* HTML attributes

### PR DESCRIPTION
**Pull request by rhcarvalho**
This is a first attempt at condensing and organizing information around phx-* in a way that fits printed A4 pages or a screenful for quick reference.

Based on forum conversation at https://elixirforum.com/t/inconsistent-naming-of-liveview-dom-patching-binding-attributes/60901/6.

### Sidebar

image

### Content

image

---

image

---

image

---

image

---

image

### CODE REVIEW

File: guides/cheatsheets/html-attrs.cheatmd [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# Special HTML Attributes
```
Comment (josevalim) [inline guides/cheatsheets/html-attrs.cheatmd:1]:
```suggestion
# `phx-` HTML attributes
```

**Comment by rhcarvalho** [comment]
I just noticed this is still pending and a few people reacted with hearts :heart: 

@josevalim do you think this is ready for merging?

**Comment by josevalim** [comment]
That’s up to the LV team. :) I am certain it will be looked into!

File: guides/cheatsheets/html-attrs.cheatmd [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+|-------------------------------------------------------------------|------------------------------------------------|
+| [`phx-feedback-for`](../client/form-bindings.md#error-feedback)   | Form field or group name                       |
+| [`phx-feedback-group`](../client/form-bindings.md#error-feedback) | Arbitrary string to identify a group of fields |
+
+### Form Example
+
+#### lib/hello_web/live/hello_live.html.heex
+
+```heex
+<form phx-change="validate" phx-submit="save" phx-auto-recover="ignore">
```
Comment (chrismccord) [inline guides/cheatsheets/html-attrs.cheatmd:88]:
```suggestion
<form phx-change="validate" phx-submit="save">
```

**Comment by chrismccord** [comment]
❤️❤️❤️🐥🔥

**Pull request merged**